lib/core/src/lib/about/about.component.stories.ts

```diff
@@ -1,106 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { Meta, moduleMetadata, StoryFn } from '@storybook/angular';
-import { AboutComponent } from './about.component';
-import { ABOUT_DIRECTIVES } from './about.module';
-import { AuthenticationService } from '../auth/services/authentication.service';
-import { AuthenticationMock } from '../auth/mock/authentication.service.mock';
```

Comment:
I don't see any usages for AuthenticationMock so looks like that mock was used only by storybook probably. Can you remove?